.npmignore

```diff
@@ -1,25 +1,13 @@
-.project
 .settings
-
-demos
-examples
-Gruntfile.js
-products
-
-src/components
-src/lib/fonts
-src/lib/jquery/plugins
-src/lib/jquery/ui
-
-src/framework/core/css
-src/framework/preferences
-
-tests/3rd-party-tests
-tests/component-tests
-tests/framework-tests
-tests/manual-tests
-tests/all-tests.html
-tests/lib/jquery-ui
-tests/lib/mockjax
-tests/lib/README
-tests/test-core/utils/js/DebugFocus.js
+.gitignore
+.npmignore
+.project
+build/
+demos/
+examples/
+products/
+tests/component-tests/
+tests/framework-tests/
+tests/manual-tests/
+tests/node-tests/
+tests/node-tests/all-tests.html
```

Comment:
we should probably also remove "tests/test-core/testTests" which are the unit tests for the IoC Testing Framework. This file should probably actually be under the jqUnit tests or somewhere similar in the framework-tests directory.